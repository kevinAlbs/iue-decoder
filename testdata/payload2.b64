Am2IfShB4k/NqP8INteqCxUCEikm46tQNyXYxnUWcZ2J7mXfvZFHvSfQwQoXgUPt9I2Q1h3aN1K4mkgOOfk7jaOGZtPRW3iVjaeRjUh9Xw3M+Q==