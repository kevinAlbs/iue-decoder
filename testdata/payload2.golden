BlobSubtype=2 (FLE1RandomEncryptedValue)
KeyUUID=6d887d2841e24fcda8ff0836d7aa0b15
OriginalBsonType=2
Ciphertext=122926e3ab503725d8c67516719d89ee65dfbd9147bd27d0c10a178143edf48d90d61dda3752b89a480e39f93b8da38666d3d15b78958da7918d487d5f0dccf9
